### PR TITLE
Hclang 52 mathmatical representation cfg

### PR DESCRIPTION
**Pull request by jones1994**
### Description
- All mathmatical syntax oriented report content and proof added to the report in the theory section.

### Changelog
Added new file to Theory.
Added new digital version of the HCL as CFL proof.

### Issues solved 
 - HCLANG-52

## FOR REVIEWER [Do not remove]
A small checklist of things to note when you are doing a review. Make sure these things apply before going into master.
 - Did the PR actually fix the issue
 - Comment anything that doesn't make sense or anything you want clarified. Many comments are always ok
 - Make sure this is ready for exams if this is going into master
 - It's always a good idea having another reviewer look at a PR as well, if it's major. get a buddy.

### Regarding filenames
- all .tex files use pascal case
- all other files [except in /code/] use snake case

### Regarding code [if applicable]
- Check [coding conventions](https://kotlinlang.org/docs/reference/coding-conventions.html)
- Are public methods documented? 
- Are there testcases for new features / methods?
- Have you tried running the source and tested things that cannot be test-cased?

### Regarding Report [if applicable]
- Does each line have a linechange in the sourcecode?
- Does sentences make sense?
- Did you do spellchecking?
- Citations?

### CODE REVIEW

File: report/3.Theory/SyntaxTheory.tex [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+This section concerns with the concise definitions of HCL's syntax.
+First the proper CFG recognizing HCL will be presented, which will be used to proof that HCL is a CFL. 
+Afterwards the most specific computational model for describing HCL will be identified and presented. 
+Lastly the token categories will be described in a computational model. 
+The conclusions of this section will provide the bases for implementing and testing the parser of the HCL compiler. 
+The majority of this section will be based on the EBNF details touched upon in the previous chapter.
+
+\subsection{HCL as an CFL}
+
+To proof that HCL is a CFL, it is necessary to construct a pushdown automaton (PDA).
```
Comment (nillas12) [inline report/3.Theory/SyntaxTheory.tex]: "To proof" til "To prove"

File: report/3.Theory/SyntaxTheory.tex [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+			The next character on the input string is read and compared to t. 
+			If matched, repeat. 
+			If not, reject the branch.
+			\item If the top of stack is the marker symbol \$, enter the accept state of the automaton. 
+			This will accept the input string.
+		\end{itemize}
+	\end{itemize}
+\end{center}
+
+To create the automaton firstly a start state needs to be added at the beginning of the automaton.
```
Comment (nillas12) [inline report/3.Theory/SyntaxTheory.tex]: Tilføj komma mellem "automaton" og "firstly". Also lyder firstly ikke lidt mærkeligt.

File: report/3.Theory/SyntaxTheory.tex [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+			\item If the top of stack is the marker symbol \$, enter the accept state of the automaton. 
+			This will accept the input string.
+		\end{itemize}
+	\end{itemize}
+\end{center}
+
+To create the automaton firstly a start state needs to be added at the beginning of the automaton. 
+Secondly a transition rule on the form $\epsilon,\epsilon->S\$$ needs to be add between the start state and a new state called the loop state.
+Here S is a variable.
+From the loop state to the finale accept state there needs to be a transition rule on the form $\epsilon,\$->\epsilon$.
```
Comment (nillas12) [inline report/3.Theory/SyntaxTheory.tex]: Change "Finale" to "Final"

File: report/3.Theory/SyntaxTheory.tex [hunk 4]
```diff
@@ -0,0 +1,178 @@
+% This document needs to include:
```
Comment (nillas12) [inline report/3.Theory/SyntaxTheory.tex]: Generelt til denne: Ensure that all abbreviations are present in the Abbreviations.tex file

File: report/3.Theory/SyntaxTheory.tex [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+To create the automaton firstly a start state needs to be added at the beginning of the automaton. 
+Secondly a transition rule on the form $\epsilon,\epsilon->S\$$ needs to be add between the start state and a new state called the loop state.
+Here S is a variable.
+From the loop state to the finale accept state there needs to be a transition rule on the form $\epsilon,\$->\epsilon$.
+To convert the EBNF's production rules to a PDA, two conversion rules needs to be followed.
+
+For each production rule on the form $A->w$, where A is a variable and w is a string the transition rule $\epsilon,A->w$ needs to be added to the automaton from the loop state back to the loop state.
+For each terminal a transition rule $a,a->\epsilon$ needs to be added to the automaton from the loop state back to the loop state.
+
+The finale non deterministic pushdown automaton (NPDA) can be seen in Appendix %Tilføj reference når lortet er i appendix.
```
Comment (nillas12) [inline report/3.Theory/SyntaxTheory.tex]: Finale til final. Er ret sikker på at Finale ikke er i engelsk (Ud over som "As the grand finale". Når det er states vil jeg mene det skal være final.

File: report/3.Theory/SyntaxTheory.tex [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+Y cannot be of the length 0.
+
+The first condition is not upheld since if $i = 0$ the new string cannot contain an equal amount of left and right parentheses because of the same reason as to why the second condition is upheld.
+
+This concludes the proof by contradiction.
+Since HCl does not uphold the required property for it to be a regular language, it it not and the only automaton capable of recognizing the language is the NDPA described in the previous sections.
+
+\subsection{Types and Identifiers}
+So far the syntax of HCL has been properly defined as a CFL. 
+Meaning that all Variables, terminals and the transition rules for recognizing HCl has been described in a computational model as seen on page
```
Comment (C0DK) [inline report/3.Theory/SyntaxTheory.tex]: Variables er med stort "V"

Comment (C0DK) [inline report/3.Theory/SyntaxTheory.tex]: HCL ikke HCl

File: report/3.Theory/SyntaxTheory.tex [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+
+The first condition is not upheld since if $i = 0$ the new string cannot contain an equal amount of left and right parentheses because of the same reason as to why the second condition is upheld.
+
+This concludes the proof by contradiction.
+Since HCl does not uphold the required property for it to be a regular language, it it not and the only automaton capable of recognizing the language is the NDPA described in the previous sections.
+
+\subsection{Types and Identifiers}
+So far the syntax of HCL has been properly defined as a CFL. 
+Meaning that all Variables, terminals and the transition rules for recognizing HCl has been described in a computational model as seen on page 
+%Indsæt sidetal for HCL CFG definition.
```
Comment (C0DK) [inline report/3.Theory/SyntaxTheory.tex]: insert noaw

File: report/3.Theory/SyntaxTheory.tex [hunk 8]
```diff
@@ -0,0 +1,178 @@
...
+and in appendix.
+%Indæst reference til appendix hvor NPDA for HCL er beskrevet.
```
Comment (C0DK) [inline report/3.Theory/SyntaxTheory.tex]: insert

File: report/3.Theory/SyntaxTheory.tex [hunk 9]
```diff
@@ -0,0 +1,178 @@
...
+An alphabet is a finite set of symbols, a string is a finite sequence of symbols and a language is a set of strings.
+To describe the languages mentioned above, the Alphabets used in the informal and formal notations needs to be defined. 
+These can be seen in Table 3.1.
+
+\begin{table}[!htb]
+	\centering
+	\begin{tabular}{|l|l|}
+		\hline
+		\textbf{Alphabet} & \textbf{Set}                            \\ \hline
+		T                 & \{true\}                                \\ \hline
```
Comment (C0DK) [inline report/3.Theory/SyntaxTheory.tex]: True - not true
False - not false

File: report/3.Theory/SyntaxTheory.tex [hunk 10]
```diff
@@ -0,0 +1,178 @@
...
+
+By using the set construction method, the languages can now be defined as shown in Table 3.2.
+
+\begin{table}[!htb]
+	\centering
+	\label{my-label}
+	\begin{tabular}{|l|l|}
+		\hline
+		\textbf{Name}     & \textbf{Set}                                    \\ \hline
+		$L_{none}$        & \{w | w $\in$ \{none\}\}                        \\ \hline
```
Comment (C0DK) [inline report/3.Theory/SyntaxTheory.tex]: None - not none

File: report/3.Theory/SyntaxTheory.tex [hunk 11]
```diff
@@ -0,0 +1,178 @@
...
+	\centering
+	\label{my-label}
+	\begin{tabular}{|l|l|}
+		\hline
+		\textbf{Name}     & \textbf{Regular Expression}         \\ \hline
+		$L_{none}$        & none                                \\ \hline
+		$L_{bool}$        & true $\cup$ false                   \\ \hline
+		$L_{number}$      & $D^+(.D^+)^*$                       \\ \hline
+		$L_{text}$        & $"C^*"$                               \\ \hline
+		$L_{identifiers}$ & $C^+(R_{eserved} \cup \epsilon)C^+$ \\ \hline
```
Comment (jones1994) [inline report/3.Theory/SyntaxTheory.tex]: Remove identifier regex

Comment (C0DK) [inline report/3.Theory/SyntaxTheory.tex]: R_eserved?
wut?

File: report/3.Theory/SyntaxTheory.tex [hunk 12]
```diff
@@ -0,0 +1,178 @@
...
+		F                 & \{false\}                               \\ \hline
+		$Z^+$             & The set of all positive integers        \\ \hline
+		$Z^-$             & The set of all negative integers        \\ \hline
+		R                 & The set of all real numbers             \\ \hline
+		D                 & \{$Z^+ \cup Z^-$\}                      \\ \hline
+		$S_p$             & The set of all special characters      \\ \hline
+		$L_l$             & The set of all lower-case letters       \\ \hline
+		$L_u$             & The set of all upper-case letters       \\ \hline
+		C                 & $S_p \cup L_l \cup L_u \cup D$          \\ \hline
+		$R_{eserved}$     & The set of all reserved keywords in HCL \\ \hline
```
Comment (C0DK) [inline report/3.Theory/SyntaxTheory.tex]: Change this to "all reserved keywords, specialchars and types"

File: report/3.Theory/SyntaxTheory.tex [hunk 13]
```diff
@@ -0,0 +1,178 @@
...
+	\centering
+	\label{my-label}
+	\begin{tabular}{|l|l|}
+		\hline
+		\textbf{Name}     & \textbf{Set}                                    \\ \hline
+		$L_{none}$        & \{w | w $\in$ \{none\}\}                        \\ \hline
+		$L_{bool}$        & \{w | w $\in$ (T $\cup$ F)\}                    \\ \hline
+		$L_{number}$      & \{w | w $\in$ R\}                               \\ \hline
+		$L_{text}$        & \{"w" | all strings from arbitrary alphabets\}  \\ \hline
+		$L_{identifiers}$ & \{w | w $\in (L_{text} - \{", "\}) \wedge w \notin R_{eserved}$\} \\ \hline
```
Comment (C0DK) [inline report/3.Theory/SyntaxTheory.tex]: write that identifiers are not regular

File: report/3.Theory/SyntaxTheory.tex [hunk 14]
```diff
@@ -0,0 +1,178 @@
...
+	\end{tabular}
+	\caption{Table containing the type and identifier alphabets described by the set builder construction}
+\end{table}
+
+Intuitively it is trivial that all of the languages defined above are regular languages.
```
Comment (C0DK) [inline report/3.Theory/SyntaxTheory.tex]: except identifiers

File: report/3.Theory/SyntaxTheory.tex [hunk 15]
```diff
@@ -0,0 +1,178 @@
...
+This is the case since a regular language cannot keep track of more than one symbol in terms of counting and iterations.
+Since all of the languages in Table 3.2 consists only of arbitrary concatenations of elements of various alphabets, this means that they are indeed regular.
```
Comment (C0DK) [inline report/3.Theory/SyntaxTheory.tex]: except identifiers

File: report/3.Theory/SyntaxTheory.tex [hunk 16]
```diff
@@ -0,0 +1,178 @@
...
+		$L_{number}$      & $D^+(.D^+)^*$                       \\ \hline
+		$L_{text}$        & $"C^*"$                               \\ \hline
+		$L_{identifiers}$ & $C^+(R_{eserved} \cup \epsilon)C^+$ \\ \hline
+	\end{tabular}
+	\caption{Regular Expressions describing the type and identifier languages}
+\end{table}
+
+In conclusion HCL is a context free language, meaning that it can be described by a non-deterministic pushdown automaton.
+A subset of HCL is non-regular, meaning that HCL cannot be a regular language.
+The token category languages are all regular languages.
```
Comment (C0DK) [inline report/3.Theory/SyntaxTheory.tex]: except identifers

File: report/3.Theory/SyntaxTheory.tex (hunk 1 above)
[the remark this answers is not in the document]
Comment (Lambcreek) [reply report/3.Theory/SyntaxTheory.tex]: Hvis denne sætning skal være med, bør den være mere generel:
"To prove that a language is a CFL"
Ellers bør vi bare sige, at vi tjekker om HCL er et CFL i dette afsnit

File: report/3.Theory/SyntaxTheory.tex (hunk 2 above)
Comment (Lambcreek) [reply report/3.Theory/SyntaxTheory.tex]: Det bør bare fjernes.

File: report/3.Theory/SyntaxTheory.tex (hunk 3 above)
Comment (Lambcreek) [reply report/3.Theory/SyntaxTheory.tex]: "Final accept" virker redundant - man kan bare skrive "accept"

File: report/3.Theory/SyntaxTheory.tex [hunk 17]
```diff
@@ -0,0 +1,178 @@
...
+For each terminal a transition rule $a,a->\epsilon$ needs to be added to the automaton from the loop state back to the loop state.
+
+The finale non deterministic pushdown automaton (NPDA) can be seen in Appendix %Tilføj reference når lortet er i appendix.
+
+HCL is therefore a CFL recognized by the NPDA in Appendix %indsæt blah blah...
+
+\subsection{Regularity of HCL}
+Now that it has been concluded that HCl is as CFL recognized by the CFG described in the previous subsections.
+It is now relevant to investigate whether HCl is a regular language. 
+If that would be the case, it would be possible to utilize finite state automatons to recognize the language.
```
Comment (Lambcreek) [inline report/3.Theory/SyntaxTheory.tex]: Automaton = ental
Automata = flertal
Automatons = ikke et ord ;)

Udover det, synes jeg at det lyder bedre med "If that is the case, it is possible to create a FSA that recognizes the language."

File: report/3.Theory/SyntaxTheory.tex [hunk 18]
```diff
@@ -0,0 +1,178 @@
...
+Since the set of regular languages is a subset of the context free languages, this might be the case.
```
Comment (Lambcreek) [inline report/3.Theory/SyntaxTheory.tex]: Tror at der skal kilde på denne sætning, før kommaet.

File: report/3.Theory/SyntaxTheory.tex (hunk 17 above)
Comment (Lambcreek) [reply report/3.Theory/SyntaxTheory.tex]: Der står "automatons" flere steder i dokumentet, så jeg siger bare her, at det skal ændres alle steder :)

File: report/3.Theory/SyntaxTheory.tex (hunk 1 above)
Comment (jones1994) [reply report/3.Theory/SyntaxTheory.tex]: Fixed

File: report/3.Theory/SyntaxTheory.tex [hunk 19]
```diff
@@ -0,0 +1,178 @@
...
+The finite state automatons are more trivial to implement than a NPDA. 
+Since the HCL language is a relatively complex language, it would be unrealistic to sequentially go through the entire language while trying to construct a finite state automaton, that recognizes HCL.
```
Comment (Lambcreek) [inline report/3.Theory/SyntaxTheory.tex]: Aldrig komma før "that"!

Also (dette er ikke noget der skal ændres): Her kalader du HCL relativt kompleks, men der er andre steder, hvor vi siger relativt simpel. De er teknisk set begge korrekte. Bare noget, som vi skal tænke over.

File: report/3.Theory/SyntaxTheory.tex (hunk 2 above)
Comment (jones1994) [reply report/3.Theory/SyntaxTheory.tex]: firstly removed and comma added

File: report/3.Theory/SyntaxTheory.tex [hunk 20]
```diff
@@ -0,0 +1,178 @@
...
+			If matched, repeat. 
+			If not, reject the branch.
+			\item If the top of stack is the marker symbol \$, enter the accept state of the automaton. 
+			This will accept the input string.
+		\end{itemize}
+	\end{itemize}
+\end{center}
+
+To create the automaton firstly a start state needs to be added at the beginning of the automaton. 
+Secondly a transition rule on the form $\epsilon,\epsilon->S\$$ needs to be add between the start state and a new state called the loop state.
```
[the remark this answers is not in the document]
Comment (jones1994) [reply report/3.Theory/SyntaxTheory.tex]: fixed

File: report/3.Theory/SyntaxTheory.tex (hunk 3 above)
Comment (jones1994) [reply report/3.Theory/SyntaxTheory.tex]: Finale removed

File: report/3.Theory/SyntaxTheory.tex (hunk 5 above)
Comment (jones1994) [reply report/3.Theory/SyntaxTheory.tex]: final removed

File: report/3.Theory/SyntaxTheory.tex [hunk 21]
```diff
@@ -0,0 +1,178 @@
...
+Here S is a variable.
+From the loop state to the finale accept state there needs to be a transition rule on the form $\epsilon,\$->\epsilon$.
+To convert the EBNF's production rules to a PDA, two conversion rules needs to be followed.
```
[the remark this answers is not in the document]
Comment (jones1994) [reply report/3.Theory/SyntaxTheory.tex]: added to abbreviations

File: report/3.Theory/SyntaxTheory.tex [hunk 22]
```diff
@@ -0,0 +1,178 @@
...
+\subsection{Regularity of HCL}
+Now that it has been concluded that HCl is as CFL recognized by the CFG described in the previous subsections.
+It is now relevant to investigate whether HCl is a regular language. 
+If that would be the case, it would be possible to utilize finite state automatons to recognize the language.
+Since the set of regular languages is a subset of the context free languages, this might be the case.
+The finite state automatons are more trivial to implement than a NPDA. 
+Since the HCL language is a relatively complex language, it would be unrealistic to sequentially go through the entire language while trying to construct a finite state automaton, that recognizes HCL.
+
+Considering the fact that the regular languages are closed under the union, concatenation and the star operations. 
+It would be sufficient to locate a subset of HCL that is not a regular language.
```
Comment (Lambcreek) [inline report/3.Theory/SyntaxTheory.tex]: Linje 61 og 62 bør være én sætning med komma imellem.
På engelsk er der komma før "and" i opremsning, så der skal stå "union, concatenation, and Kleene star...."

File: report/3.Theory/SyntaxTheory.tex (hunk 5 above)
[the remark this answers is not in the document]
Comment (jones1994) [reply report/3.Theory/SyntaxTheory.tex]: added

File: report/3.Theory/SyntaxTheory.tex [hunk 23]
```diff
@@ -0,0 +1,178 @@
...
+Here S is a variable.
+From the loop state to the finale accept state there needs to be a transition rule on the form $\epsilon,\$->\epsilon$.
+To convert the EBNF's production rules to a PDA, two conversion rules needs to be followed.
+
+For each production rule on the form $A->w$, where A is a variable and w is a string the transition rule $\epsilon,A->w$ needs to be added to the automaton from the loop state back to the loop state.
+For each terminal a transition rule $a,a->\epsilon$ needs to be added to the automaton from the loop state back to the loop state.
+
+The finale non deterministic pushdown automaton (NPDA) can be seen in Appendix %Tilføj reference når lortet er i appendix.
+
+HCL is therefore a CFL recognized by the NPDA in Appendix %indsæt blah blah...
```
[the remark this answers is not in the document]
Comment (jones1994) [reply report/3.Theory/SyntaxTheory.tex]: removed and replaced with sentence where no reference is needed

File: report/3.Theory/SyntaxTheory.tex [hunk 24]
```diff
@@ -0,0 +1,178 @@
...
+
+\subsection{Regularity of HCL}
+Now that it has been concluded that HCl is as CFL recognized by the CFG described in the previous subsections.
```
[the remark this answers is not in the document]
Comment (jones1994) [reply report/3.Theory/SyntaxTheory.tex]: fixed

File: report/3.Theory/SyntaxTheory.tex [hunk 25]
```diff
@@ -0,0 +1,178 @@
...
+It is now relevant to investigate whether HCl is a regular language.
```
[the remark this answers is not in the document]
Comment (jones1994) [reply report/3.Theory/SyntaxTheory.tex]: fixed

File: report/3.Theory/SyntaxTheory.tex [hunk 26]
```diff
@@ -0,0 +1,178 @@
...
+Now that it has been concluded that HCl is as CFL recognized by the CFG described in the previous subsections.
+It is now relevant to investigate whether HCl is a regular language. 
+If that would be the case, it would be possible to utilize finite state automatons to recognize the language.
+Since the set of regular languages is a subset of the context free languages, this might be the case.
+The finite state automatons are more trivial to implement than a NPDA. 
+Since the HCL language is a relatively complex language, it would be unrealistic to sequentially go through the entire language while trying to construct a finite state automaton, that recognizes HCL.
+
+Considering the fact that the regular languages are closed under the union, concatenation and the star operations. 
+It would be sufficient to locate a subset of HCL that is not a regular language.
+The transition rules described in the EBNF, state that a subset of HCL might consists of an arbitrary number of equal amounts of left and right parentheses.
```
Comment (Lambcreek) [inline report/3.Theory/SyntaxTheory.tex]: Consists skal være consist

File: report/3.Theory/SyntaxTheory.tex (hunk 17 above)
Comment (jones1994) [reply report/3.Theory/SyntaxTheory.tex]: Fixed sentence and looked through the section to handle the non intentional plural automaton words

File: report/3.Theory/SyntaxTheory.tex (hunk 18 above)
Comment (jones1994) [reply report/3.Theory/SyntaxTheory.tex]: Det er common knowledge fra bogen så det behøver vi ikke :)

File: report/3.Theory/SyntaxTheory.tex [hunk 27]
```diff
@@ -0,0 +1,178 @@
...
+
+This means that the third condition is upheld since $|xy|$ cannot be greater than p.
+The second condition is also upheld since y consist of at least some left parentheses. 
+If the string does contain right parentheses, which it does. 
+Y cannot be of the length 0.
+
+The first condition is not upheld since if $i = 0$ the new string cannot contain an equal amount of left and right parentheses because of the same reason as to why the second condition is upheld.
+
+This concludes the proof by contradiction.
+Since HCl does not uphold the required property for it to be a regular language, it it not and the only automaton capable of recognizing the language is the NDPA described in the previous sections.
```
[the remark this answers is not in the document]
Comment (jones1994) [reply report/3.Theory/SyntaxTheory.tex]: fixed

File: report/3.Theory/SyntaxTheory.tex (hunk 6 above)
Comment (jones1994) [reply report/3.Theory/SyntaxTheory.tex]: fixed

Comment (jones1994) [reply report/3.Theory/SyntaxTheory.tex]: fixed

File: report/3.Theory/SyntaxTheory.tex (hunk 7 above)
Comment (jones1994) [reply report/3.Theory/SyntaxTheory.tex]: removed the sentence

File: report/3.Theory/SyntaxTheory.tex (hunk 8 above)
Comment (jones1994) [reply report/3.Theory/SyntaxTheory.tex]: inserted

File: report/3.Theory/SyntaxTheory.tex (hunk 9 above)
Comment (jones1994) [reply report/3.Theory/SyntaxTheory.tex]: fixed

File: report/3.Theory/SyntaxTheory.tex (hunk 12 above)
Comment (jones1994) [reply report/3.Theory/SyntaxTheory.tex]: fixed

File: report/3.Theory/SyntaxTheory.tex (hunk 10 above)
Comment (jones1994) [reply report/3.Theory/SyntaxTheory.tex]: fixed

File: report/3.Theory/SyntaxTheory.tex [hunk 28]
```diff
@@ -0,0 +1,178 @@
...
+First the division is chosen for s.
+$s = xyz$, where $s = (((...b)))...$.
+Meaning that x consists only of left parentheses, y consists of a concatenation of left parentheses and elements from P and finally z consists only of right parentheses.
+
+The length of the string is $p < |s| < (p3)/2$, since $s = a^nbc^n$, where $n = p/2$ and $0 < |b| \leq p/2$.
+
+This means that the third condition is upheld since $|xy|$ cannot be greater than p.
+The second condition is also upheld since y consist of at least some left parentheses. 
+If the string does contain right parentheses, which it does. 
+Y cannot be of the length 0.
```
Comment (Lambcreek) [inline report/3.Theory/SyntaxTheory.tex]: Linje 90 og 91 bør være én sætning, separeret med et komma.

"If the string does contain" bør være "If the string contains"

File: report/3.Theory/SyntaxTheory.tex (hunk 13 above)
Comment (jones1994) [reply report/3.Theory/SyntaxTheory.tex]: fixed

File: report/3.Theory/SyntaxTheory.tex (hunk 14 above)
Comment (jones1994) [reply report/3.Theory/SyntaxTheory.tex]: fixed

File: report/3.Theory/SyntaxTheory.tex (hunk 15 above)
Comment (jones1994) [reply report/3.Theory/SyntaxTheory.tex]: fixed

File: report/3.Theory/SyntaxTheory.tex (hunk 11 above)
Comment (jones1994) [reply report/3.Theory/SyntaxTheory.tex]: done

Comment (jones1994) [reply report/3.Theory/SyntaxTheory.tex]: it is consistent so we will stick with it

File: report/3.Theory/SyntaxTheory.tex (hunk 27 above)
[the remark this answers is not in the document]
Comment (Lambcreek) [reply report/3.Theory/SyntaxTheory.tex]: Denne sætning lyder meget underlig. Jeg forstår godt hvad du mener, men det kan skrives meget bedre.
Also: "it it"

File: report/3.Theory/SyntaxTheory.tex (hunk 9 above)
Comment (Lambcreek) [reply report/3.Theory/SyntaxTheory.tex]: Tror at casper mener, at vi skriver med stort forbogstav til boolean literals.

File: report/3.Theory/SyntaxTheory.tex (hunk 10 above)
Comment (Lambcreek) [reply report/3.Theory/SyntaxTheory.tex]: Igen, tror at han mener at forbogstavet skal stå med stort

File: report/3.Theory/SyntaxTheory.tex (hunk 4 above)
Comment (jones1994) [reply report/3.Theory/SyntaxTheory.tex]: All should be added to abbreviations now

**Pull request merged**